### DISCUSSION TITLE
Language Server and editor extensions

### DISCUSSION THREAD

**Discussion by elf-pavlik** [LinkedSoftwareDependencies/Components.js#136] [Ideas]
Does anyone know of existing work or would be interested in collaborating on developing a [language server](https://microsoft.github.io/language-server-protocol/) that various code editors could use?

We could consider a handful of simple features that could improve the DX, jump to the definition of a class from the configuration, and provide a quicker way of searching for occurrences of the same instance, etc.

**Comment by rubensworks** [comment]
This has been on the wishlist for a very long time :-) (#13)

I heard @ajuvercr might be interested in this given his experience with building a JSON-LD language server: https://github.com/ajuvercr/jsonld-lsp
